### PR TITLE
(AB-1948159) Add change-set-specific PR Templates

### PR DESCRIPTION
**Pull request by michaeltlombardi**
# PR Summary

Prior to this commit, the only available PR Template for the project is a catch-all template designed to accommodate every workflow and change type.

This commit introduces three new PR Templates for specific change-sets: conceptual, reference, and repository documentation.

They have specific instructions tailored to those change-sets and can be used for automation in the future, especially around the task lists.

These templates will not be displayed by default when a PR is submitted; they can only be used via a url query suffix which specifies the template, such as `?quick_pull=1&template=repodocs.md`.

Usage documentation and automation tailored to these templates will need to be addressed in future commits and items.

- Resolves AB#1948159

## PR Context

Check the boxes below to indicate the content affected by this PR.

**Repository or docset configuration**
- [x] Repo documentation and configuration (.git/.github/.vscode etc.)
- [ ] Docs build files (.openpublishing.* and build scripts)
- [ ] Docset configuration (docfx.json, mapping, bread, module folder)

**Conceptual documentation**
- [ ] Files in docs-conceptual

**Cmdlet reference & about_ topics**
When changing **cmdlet reference** or **about_ topics**, the changes should be copied to all
relevant versions. Check the boxes below to indicate the versions affected by this change.

- [ ] Preview content
- [ ] Version 7.2 content
- [ ] Version 7.1 content
- [ ] Version 7.0 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _main_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [ ] This PR is ready to merge and is not **Work in Progress**. If the PR is work in progress,
  please add the prefix `WIP:` or `[WIP]` to the beginning of the title and remove the prefix when
  the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/repodocs.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  required steps and marked your PR as **Ready to Merge**.
+
+You can **always** request a review at any stage in your authoring process, the docs team is here to
+help! You do not need to submit a fully polished and finished draft; the docs team can help you get
+content ready for merge.
+
+While reviewing your PR, the docs team may make suggestions, write comments, and ask questions. When
+all requirements are satisfied, the docs team marks your PR as **Approved** and merges it. Once your
+PR is merged, it is included the next time the documentation is published. For this project, the
+documentation is published daily at 3pm Pacific Standard Time (PST).
```
Comment (sdwheeler) [inline .github/PULL_REQUEST_TEMPLATE/repodocs.md:60]:
```suggestion
## Expectations

The docs team begins to review your PR if you [request them to][gh-review-request] or if your PR
meets these conditions:

- It is not a [draft PR][gh-drafts].
- It does not have a `WIP` prefix in the title.
- It passes validation and build steps.
- It does not have any merge conflicts.
- You have checked every box in the [PR Checklist](#pr-checklist), indicating you have completed all
  required steps and marked your PR as **Ready to Merge**.

You can **always** request a review at any stage in your authoring process, the docs team is here to
help! You do not need to submit a fully polished and finished draft; the docs team can help you get
content ready for merge.

For more information about how we review PRs, please read our [editor's checklist][contrib-checklist] 
and process for [managing pull requests][contrib-managing-prs].

While reviewing your PR, the docs team may make suggestions, write comments, and ask questions. When
all requirements are satisfied, the docs team marks your PR as **Approved** and merges it. Once your
PR is merged, the changes are included the next time the documentation is published. For this
project, the documentation is published Monday-Friday at 3pm Pacific Time (PT).
```

File: .github/PULL_REQUEST_TEMPLATE/conceptual.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  required steps and marked your PR as **Ready to Merge**.
+
+You can **always** request a review at any stage in your authoring process, the docs team is here to
+help! You do not need to submit a fully polished and finished draft; the docs team can help you get
+content ready for merge.
+
+While reviewing your PR, the docs team may make suggestions, write comments, and ask questions. When
+all requirements are satisfied, the docs team marks your PR as **Approved** and merges it. Once your
+PR is merged, it is included the next time the documentation is published. For this project, the
+documentation is published daily at 3pm Pacific Standard Time (PST).
```
Comment (sdwheeler) [inline .github/PULL_REQUEST_TEMPLATE/conceptual.md:55]:
```suggestion
## Expectations

The docs team begins to review your PR if you [request them to][gh-review-request] or if your PR
meets these conditions:

- It is not a [draft PR][gh-drafts].
- It does not have a `WIP` prefix in the title.
- It passes validation and build steps.
- It does not have any merge conflicts.
- You have checked every box in the [PR Checklist](#pr-checklist), indicating you have completed all
  required steps and marked your PR as **Ready to Merge**.

You can **always** request a review at any stage in your authoring process, the docs team is here to
help! You do not need to submit a fully polished and finished draft; the docs team can help you get
content ready for merge.

For more information about how we review PRs, please read our [editor's checklist][contrib-checklist] 
and process for [managing pull requests][contrib-managing-prs].

While reviewing your PR, the docs team may make suggestions, write comments, and ask questions. When
all requirements are satisfied, the docs team marks your PR as **Approved** and merges it. Once your
PR is merged, the changes are included the next time the documentation is published. For this
project, the documentation is published Monday-Friday at 3pm Pacific Time (PT).
```

File: .github/PULL_REQUEST_TEMPLATE/reference.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+You can **always** request a review at any stage in your authoring process, the docs team is here to
+help! You do not need to submit a fully polished and finished draft; the docs team can help you get
+content ready for merge.
+
+While reviewing your PR, the docs team may make suggestions, write comments, and ask questions. When
+all requirements are satisfied, the docs team marks your PR as **Approved** and merges it. Once your
+PR is merged, it is included the next time the documentation is published. For this project, the
+documentation is published daily at 3pm Pacific Standard Time (PST).
+
```
Comment (sdwheeler) [inline .github/PULL_REQUEST_TEMPLATE/reference.md:66]:
```suggestion
## Expectations

The docs team begins to review your PR if you [request them to][gh-review-request] or if your PR
meets these conditions:

- It is not a [draft PR][gh-drafts].
- It does not have a `WIP` prefix in the title.
- It passes validation and build steps.
- It does not have any merge conflicts.
- You have checked every box in the [PR Checklist](#pr-checklist), indicating you have completed all
  required steps and marked your PR as **Ready to Merge**.

You can **always** request a review at any stage in your authoring process, the docs team is here to
help! You do not need to submit a fully polished and finished draft; the docs team can help you get
content ready for merge.

For more information about how we review PRs, please read our [editor's checklist][contrib-checklist] 
and process for [managing pull requests][contrib-managing-prs].

While reviewing your PR, the docs team may make suggestions, write comments, and ask questions. When
all requirements are satisfied, the docs team marks your PR as **Approved** and merges it. Once your
PR is merged, the changes are included the next time the documentation is published. For this
project, the documentation is published Monday-Friday at 3pm Pacific Time (PT).
```

**Review by sdwheeler — CHANGES_REQUESTED** [review]
See my suggested edits.

For discussion:
- what is the value of separating conceptual and reference into separate templates? Many PR include both.
- How will we get users to use these templates if they are not the default template?
- Who is going to use the repodocs.md template? How? Why? If it is automation driven usage, should we just put the template in automation rather than here?

**Comment by michaeltlombardi** [comment]
Discussion with @sdwheeler led to an alternate plan for templates in light of the UX difficulties around using multiple templates in GitHub at this time.

We will instead simplify the default template and invest in PR automation to accomplish the same goals.

In the future, if the implementation of multiple PR templates for GitHub improves to the point where the UX is delightful, we can iterate on our approach again.

**Pull request closed without merging**